.eslintrc.yml

```diff
@@ -15,18 +16,8 @@ plugins:
 root: true
 
 rules:
-  security/detect-buffer-noassert: warn
-  security/detect-child-process: warn
-  security/detect-disable-mustache-escape: warn
-  security/detect-eval-with-expression: warn
-  security/detect-new-buffer: warn
-  security/detect-no-csrf-before-method-override: warn
-  security/detect-non-literal-fs-filename: warn
-  security/detect-non-literal-regexp: warn
-  security/detect-non-literal-require: warn
-  security/detect-possible-timing-attacks: warn
-  security/detect-pseudoRandomBytes: warn
-  security/detect-unsafe-regex: warn
```

Comment:
All these default rules are enabled in _"plugin:security/recommended"_ now (as of v1.4.0).